src/index.js

```diff
@@ -475,62 +495,250 @@ const adaptive_test_node = {
 };
 
 // trials to add gamification
-const preload_intro_video = {
+const images = [
+  left_arrow_image,
+  right_arrow_image,
+];
+
+const videos = [
+  intro_video_1,
+  intro_video_2,
+  intro_video_3,
+  game_instructions_1,
+  game_instructions_2,
+  game_instructions_3,
+  pos_instuctions_feedback,
+  neg_instuctions_feedback,
+  two_back_instructions,
+  three_back_instructions,
+  fix_robot_1,
+  fix_robot_2,
+  fix_robot_3,
+  fix_robot_4,
+  fix_robot_5,
+  fix_robot_6,
+  fix_robot_n,
+  generic_game_break,
+];
+
+const preload_videos = {
   type: jsPsychPreload,
-  video: intro_video,
+  video: videos,
 };
 
-const exit_fullscreen = {
-  type: jsPsychFullScreen,
-  fullscreen_mode: false,
-  delay_after: 0,
+const preload_images = {
+  type: jsPsychPreload,
+  image: images,
+};
+
+const video_parameters = {
+  type: videoKeyboardResponse,
+  trial_ends_after_video: false,
+  response_allowed_while_playing: true,
+  trial_duration: null,
+  choices: "ALL_KEYS",
+  width: VIDEO_WIDTH,
+  height: VIDEO_HEIGHT,
+};
+
+const intro_video_node_1 = {
+  stimulus: [intro_video_1],
+  ...video_parameters,
+};
+
+const intro_video_node_2 = {
+  stimulus: [intro_video_2],
+  ...video_parameters,
+};
+
+const intro_video_node_3 = {
+  stimulus: [intro_video_3],
+  ...video_parameters,
+};
+
+// trying to figure out how to make it so that this will push only after the neg_instructions_fb
+// is pushed (and makes sure that they get it right )
+const neg_instuctions_feedback_parameters = {
+  type: videoKeyboardResponse,
+  trial_ends_after_video: false,
+  response_allowed_while_playing: true,
+  trial_duration: null,
+  choices: [CORRECT_KEY_PRESS, WRONG_KEY_PRESS],
+  width: VIDEO_WIDTH,
+  height: VIDEO_HEIGHT,
+  stimulus: () => {
+    const last_trial_correct = jsPsych.data.get().last(1).values()[0].correct;
+    if (last_trial_correct) {
+      // the parameter value has to be returned from the function
+      return [""];
+    }
+    // the parameter value has to be returned from the function
+    return [neg_instuctions_feedback];
+  },
 };
 
-const intro_video_node = {
+const game_instructions_feedback_trial = {
   type: videoKeyboardResponse,
-  stimulus: [intro_video],
   trial_ends_after_video: true,
   response_allowed_while_playing: true,
   trial_duration: null,
-  choices: 'NO_KEYS ',
+  choices: "ALL_KEYS",
+  width: VIDEO_WIDTH,
+  height: VIDEO_HEIGHT,
+  stimulus: () => {
+    const last_trial_correct = jsPsych.data.get().last(1).values()[0].correct;
+    if (last_trial_correct) {
+      return [pos_instuctions_feedback];
+    }
+    return [neg_instuctions_feedback];
+  },
+};
+
+const game_instructions = {
+  stimulus: [game_instructions_1],
+  type: videoKeyboardResponse,
+  trial_ends_after_video: false,
+  response_allowed_while_playing: true,
+  trial_duration: null,
+  width: 1440,
+  height: 1800,
+  data: {
+    task: "practice_response",
+    correct_response: "CORRECT_KEY_PRESS",
```

Comment:
Do you mean to do

```
correct_response: CORRECT_KEY_PRESS,
```

That is, without the quotes? The quotes make a big difference because one is a string and the other is the value in the string.